docs/DeveloperGuide.md

```diff
@@ -300,30 +300,93 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `UGTeach` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: Adding a student**
 
 **MSS**
+1. User enters command to create new student entry.
+2. System displays `[Student's name] successfully added!` and command line is cleared.
```

Comment:
Hi @Soumyadip-cmd , would it be better to change this to: "System displays success message ....." so that UI specific details like success message content is not included?

Reply:
Thank you for the review. I have made changes following your advice.